test/spec/collection-management/clustered-indexes.json

```diff
@@ -1,9 +1,10 @@
 {
   "description": "clustered-indexes",
-  "schemaVersion": "1.0",
+  "schemaVersion": "1.4",
```

Comment:
Changes from https://github.com/mongodb/specifications/commit/d1458823bd810014df9da16d3a5354d2269ab865 are also included in this PR.  That commit is mentioned in [NODE-4210](https://jira.mongodb.org/browse/NODE-4210), but since we've already completed the clustered index work I decided to pull these changes in.

Technically they're unnecessary until Node tackles NODE-3735, so I'm happy to revert the changes to `clustered-indexes.*` for now.